lib/cc_plugin_imos/cc_plugin_imos/util.py

```diff
@@ -189,6 +190,7 @@ def find_data_variables(dataset, coordinate_variables, ancillary_variables):
         if var not in coordinate_variables and var not in \
             ancillary_variables and var.dimensions and var not in \
             auxiliary_coordinate_variables \
+            and not hasattr(var, 'flag_meanings') \
```

Comment:
Yes, that looks ok. It would be nice to separate this change (which also affects the IMOS checkers) from your addition of the SRS checker. Could you put it in a separate commit? (if not a separate PR)